### ISSUE TITLE
Edit button for Maker Checker tasks goes out of borders

### ISSUE THREAD

**Issue by edcable** [openMF/community-app#1773]
See https://demo.openmf.org/reskin/#/admin/viewmctasks

<img width="1157" alt="screen shot 2016-12-27 at 2 50 14 pm" src="https://cloud.githubusercontent.com/assets/823454/21510059/24513278-cc44-11e6-81d8-db854d7e1901.png">

**Comment by skprabhanjan** [comment]
I would like to work on this bug, Can you please walk me through the specific details and about the label "reskin" because in the source code available the bug isnt reflected.

**Comment by nazeer1100126** [comment]
Checkout branch 'new_reskin' and run the application. 
Prerequisites: You need to install Ruby 2.x to build and run this application.

**Comment by skprabhanjan** [comment]
I seemed to have figured it out and fixed it. 
![screen shot 2017-02-03 at 9 02 29 pm](https://cloud.githubusercontent.com/assets/17094832/22597073/2669ef2c-ea54-11e6-8efa-bb78494d6350.png)

Can you please provide me a way forward to testing this and checking the feasibility.?

**Comment by skprabhanjan** [comment]
@mbj36 , I have created a new PR for this issue, Please review:)

**Comment by mbj36** [comment]
fixed
